src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -685,8 +685,8 @@ private static DependencyMode toDependencyMode(DependencyModeFlag flag) {
         name = "--language_out",
         usage =
             "Sets the language spec to which output should conform. "
-                + "Options: ECMASCRIPT3, ECMASCRIPT5, ECMASCRIPT5_STRICT, "
-                + "ECMASCRIPT_2015, STABLE")
+                + "Options: ECMASCRIPT3, ECMASCRIPT5, ECMASCRIPT5_STRICT, ECMASCRIPT6, "
```

Comment:
Since ES6 is just an old synonym for ES2015, should we omit that one here?

Reply:
Since the requested changes are trivial, I'll import the PR and do the fix internally.